src/transformers/image_utils.py

```diff
@@ -116,6 +116,20 @@ def to_pil_image(self, image, rescale=None):
             return PIL.Image.fromarray(image)
         return image
 
+    def convert_rgb(self, image):
+        """
+        Converts `image` to RGB format. Note that this will trigger a conversion of `image` to a PIL Image.
```

Comment:
The docstring doesn't say that, in case an image is a PyTorch tensor or NumPy array, then the same thing is returned. There's no conversion to PIL happening.

Reply:
Fixed it!